### PR TITLE
docs: add Stanimira Vlaeva to contributors

### PR DESCRIPTION
**Pull request by sis0k0**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [X] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

This PR adds [Stanimira Vlaeva](https://twitter.com/StanimiraVlaeva) to the list of Angular GDEs. -> cc @StephenFluin

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/marketing/contributors.json [hunk 1]
```diff
@@ -699,12 +699,22 @@
     "bio": "Brian is a software engineer and GDE in Angular with a passion for learning, writing, speaking, teaching and mentoring. Brian has been building web applications for over 20 years and has long been a fanboy of JavaScript. When not in front of his Macbook Pro Brian is in the Rocky Mountains skiing or hiking.",
     "group": "GDE"
   },
+
```
Comment (StephenFluin) [inline aio/content/marketing/contributors.json]: Can we remove this extra whitespace change?

**Review by StephenFluin — CHANGES_REQUESTED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (gkalpak) [reply aio/content/marketing/contributors.json]: Whyyyyy? 😢 It is making whitespace consistent with the rest of the file. 👌 
I whole:heart:edly approve of this whitespace change (and frown upon @jeffbcross's introducing the inconsistency in the first place 👀)

Comment (gkalpak) [reply aio/content/marketing/contributors.json]: Ideally, we should get @jeffbcross to submit a PR with the fix 🦈

**Review by StephenFluin — APPROVED** [review]

File: aio/content/marketing/contributors.json (hunk 1 above)
Comment (StephenFluin) [reply aio/content/marketing/contributors.json]: I can be onboard with fixing it here. I just have an initial allergic reaction to unrelated changes. :)

Comment (jeffbcross) [reply aio/content/marketing/contributors.json]: How dare you question my whitespace judgment, @gkalpak

Comment (gkalpak) [reply aio/content/marketing/contributors.json]: Can't read your comment. The test is blurry. I think it's because I'm still frowning at you 😒

**Review by brandonroberts — APPROVED** [review]

**Comment by brandonroberts** [comment]
Caretaker: adding a global approval

**Review by IgorMinar — APPROVED** [review]

**Pull request closed without merging**